customers/customer.go

```diff
@@ -33,6 +51,11 @@ type (
 		StatusResponse *checkout.StatusResponse `json:"api_response,omitempty"`
 	}
 
+	GetCustomerResponse struct {
```

Comment:
This should just be `GetResponse` I believe, following convention since we're already in the `customer` package